### PR TITLE
Add getLogoutUrlFromSessionCookie helper method

### PR DESCRIPTION
**Pull request by mattgd**
## Description
Add getLogoutUrlFromSessionCookie helper method.

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[x] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

**Review by chantastic — COMMENTED** [review]
Whoa! You're fast.
Love seeing the coverage here as well. Big fan. Let it rip!

**Review by chantastic — APPROVED** [review]
Whoa! You're fast.
Love seeing the coverage here as well. Big fan. Let it rip!

File: src/user-management/user-management.ts [hunk 1]
```diff
@@ -961,6 +961,23 @@ export class UserManagement {
...
+    cookiePassword = process.env.WORKOS_COOKIE_PASSWORD,
+  }: SessionHandlerOptions): Promise<string> {
+    const authenticationResponse = await this.authenticateWithSessionCookie({
+      sessionData,
+      cookiePassword,
+    });
+
+    if (!authenticationResponse.authenticated) {
+      const { reason } = authenticationResponse;
+      throw new Error(`Failed to authenticate session: ${reason}`);
```
Comment (PaulAsjes) [inline src/user-management/user-management.ts:975]: This error is weird. I'd be surprised and confused if I tried to get a log out URL and was given a "failed to authenticate" error. How about 

```suggestion
      throw new Error(`Failed to extract session ID for logout URL: ${reason}`);
```

File: src/user-management/user-management.ts [hunk 2]
```diff
@@ -961,6 +961,23 @@ export class UserManagement {
...
+    }
+
+    return `${this.workos.baseURL}/user_management/sessions/logout?session_id=${authenticationResponse.sessionId}`;
```
Comment (PaulAsjes) [inline src/user-management/user-management.ts:978]: Why not call `getLogoutUrl()` here?

File: src/user-management/user-management.ts [hunk 3]
```diff
@@ -961,6 +961,23 @@ export class UserManagement {
     return `${this.workos.baseURL}/user_management/sessions/logout?session_id=${sessionId}`;
   }
 
+  async getLogoutUrlFromSessionCookie({
```
Comment (PaulAsjes) [inline src/user-management/user-management.ts:964]: This could use a JSDoc style comment on when you'd use this over `getLogoutUrl`.

**Review by PaulAsjes — CHANGES_REQUESTED** [review]

File: src/user-management/user-management.ts (hunk 2 above)
Comment (mattgd) [reply src/user-management/user-management.ts:978]: I should be, fixing!

File: src/user-management/user-management.ts (hunk 1 above)
Comment (mattgd) [reply src/user-management/user-management.ts:975]: Makes sense, updating!

File: src/user-management/user-management.ts (hunk 3 above)
Comment (mattgd) [reply src/user-management/user-management.ts:964]: Adding!

**Review by PaulAsjes — APPROVED** [review]

**Pull request merged**